api/v0/proto/blob_service.proto

```diff
@@ -0,0 +1,101 @@
+syntax = "proto3";
+
+package celestia.node.v1.blob;
```

Comment:
nit: we should follow buf lint rules. package should follow folder path. It would be good to also set up protolint in order to adhere to proto standards and make sure things are consistent